docs/site/Memory-connector.md

```diff
@@ -94,32 +81,37 @@ using the memory connector exits, all model instances are lost. To maintain data
 across application restarts, specify a JSON file in which to store the data with
 the `file` property when creating the data source.
 
-The simplest way to do this is by editing `server/datasources.json`; for
-example:
+The simplest way to do this is by editing
+`src/datasources/[datasource name].datasource.ts`; for example:
 
-{% include code-caption.html content="server/datasources.json" %}
+{% include code-caption.html content="src/datasources/db.datasource.ts" %}
 
-```javascript
-{
-  "db": {
-    "name": "db",
-    "connector": "memory",
-    "file": "mydata.json"
-  }
-}
-```
+```diff
+import {inject} from '@loopback/core';
+import {juggler} from '@loopback/repository';
 
-You can also set the persistence file in a boot script; for example:
+const config = {
+  name: 'db',
+  connector: 'memory',
+  localStorage: '',
++ file: './data/db.json',
+- file: '',
+};
 
-{% include code-caption.html content="server/boot/script.js" %}
+export class DbDataSource extends juggler.DataSource {
+  static dataSourceName = 'db';
+  static readonly defaultConfig = config;
 
-```javascript
-var memory = loopback.createDataSource({
-  connector: loopback.Memory,
-  file: 'mydata.json',
-});
+  constructor(
+    @inject('datasources.config.db', {optional: true})
+    dsConfig: object = config,
+  ) {
+    super(dsConfig);
+  }
+}
 ```
 
-When the application exits, the memory connector will then store data in the
-`mydata.json` file, and when it restarts will load the saved data from that
-file.
+## Limitations
```

Comment:
Let's move the list of limitations to the top, after the list of the supported features.